test/test_pipeline/components/regression/test_ridge_regression.py

```diff
@@ -11,11 +11,12 @@ class RidgeComponentTest(BaseRegressionComponentTest):
     res = dict()
     res["default_boston"] = 0.7035465377559671
     res["default_boston_iterative"] = None
-    res["default_boston_sparse"] = 0.1086549402189615
+    res["default_boston_sparse"] = 0.1163004000785135
+    res["default_boston_sparse_places"] = 5
     res["default_boston_iterative_sparse"] = None
     res["default_diabetes"] = 0.32614416980439365
     res["default_diabetes_iterative"] = None
-    res["default_diabetes_sparse"] = 0.1298841767742679
+    res["default_diabetes_sparse"] = 0.12989753681434824
```

Comment:
Apparently ridge regression had a small bug when dealing with X sparse. It has been fixed in v0.22.
Model details here: https://github.com/scikit-learn/scikit-learn/pull/13995